admin/common_auth/views.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth import authenticate, logout as logout_user, login as auth_login
+from django.shortcuts import render, redirect
+
+from forms import LoginForm
+
+def login(request):
+    if request.user.is_authenticated():
+        return redirect('/admin/')
```

Comment:
You should use the named route for admin here. If they ever change the url of admin you don't want your name next to the blame line.